### PR TITLE
fix: add profile arpanaditya

### PR DESCRIPTION
**Pull request by arpanaditya**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/981"></a>

### CODE REVIEW

**Comment by arpanaditya** [comment]
I have created my profile. Please add it to the linkfree project

File: public/data/arpanaditya.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    {
+      "title": "First Open-source contribution",
+      "image": "https://github.com/arpanaditya.png",
+      "date": "Dec 2020",
+      "icon": "github",
+      "color": "grey",
+      "description": "Started contributing to different open-source projects and communities.",
+      "url": "https://github.com/arpanaditya"
+  ]
```
Comment (Panquesito7) [inline public/data/arpanaditya.json:44]:
```suggestion
  ],
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/arpanaditya.json (hunk 1 above)
Comment (arpanaditya) [reply public/data/arpanaditya.json]: thank you

**Comment by arpanaditya** [comment]
why the jobs are failing @Panquesito7

**Review by arpanaditya — COMMENTED** [review]
Thank you @Panquesito7

File: public/data/arpanaditya.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    {
+      "title": "First Open-source contribution",
+      "image": "https://github.com/arpanaditya.png",
+      "date": "Dec 2020",
+      "icon": "github",
+      "color": "grey",
+      "description": "Started contributing to different open-source projects and communities.",
+      "url": "https://github.com/arpanaditya"
+  ],
```
Comment (krishguptadev) [inline public/data/arpanaditya.json:44]:
```suggestion
  ]
```

File: public/data/arpanaditya.json (hunk 1 above)
Comment (krishguptadev) [inline public/data/arpanaditya.json:44]:
```suggestion
    }
  ]
```

**Review by krishguptadev — APPROVED** [review]

**Review by eddiejaoude — APPROVED** [review]
Great collaboration

**Comment by arpanaditya** [comment]
Thank you @eddiejaoude @Panquesito7 @krishguptadev  for your help

**Pull request merged**